src/mpid/ch4/generic/mpidig_init.c

```diff
@@ -44,8 +44,8 @@ int MPIDIG_init(void)
 #endif
 
     MPIDI_global.cmpl_list = NULL;
-    OPA_store_int(&MPIDI_global.exp_seq_no, 0);
-    OPA_store_int(&MPIDI_global.nxt_seq_no, 0);
+    MPL_atomic_release_store_uint64(&MPIDI_global.exp_seq_no, 0);
+    MPL_atomic_release_store_uint64(&MPIDI_global.nxt_seq_no, 0);
 
```

Comment:
Since generally we should always use the stronger semantics of `acquire_load` and `release_store`, I have introduced wrapper macros such as `MPL_atomic_store_uint64`: https://github.com/pmodels/mpich/pull/4199/commits/5b41296958e6a32bfe170074de73fdd388526a02, for simpler and better readability.

Reply:
Makes sense to me. I sort of remembered that PR, but didn't find it in a (very limited) search.